src/stories/plans/stepsTable.stories.tsx

```diff
@@ -27,71 +27,56 @@ const sampleJobs: { [key: string]: Job } = {
   Complete: sampleJob2,
   Failed: sampleJob3,
 };
-const samplePreflights: { [key: string]: Preflight } = {
+const samplePreflights: { [key: string]: Preflight | null } = {
+  None: null,
```

Comment:
I went ahead and combined the various Preflight examples into one story, so now we have one for pre-install and one for installation.